"dotnet" exited with code -2147450730

I cannot build latest master locally. I also have the same problem when trying to build any project based on the nuget package (e.g. [Solidity.Roslyn](https://github.com/Pzixel/Solidity.Roslyn))
```
1>------ Build started: Project: CodeGeneration.Roslyn.Tests, Configuration: Debug Any CPU ------
1>C:\Users\pzixe\Documents\Repos\CodeGeneration.Roslyn\src\CodeGeneration.Roslyn.Tasks\build\CodeGeneration.Roslyn.BuildTime.targets(20,5): error MSB6006: "dotnet" exited with code -2147450730.
1>Done building project "CodeGeneration.Roslyn.Tests.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 7 up-to-date, 0 skipped ==========
```

Complete detailed log: https://gist.github.com/Pzixel/59c93760658f92ccb49908186cd76020

--- a/src/CodeGeneration.Roslyn.Tests/CodeGeneration.Roslyn.Tests.csproj
+++ b/src/CodeGeneration.Roslyn.Tests/CodeGeneration.Roslyn.Tests.csproj
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <TargetFramework>net462</TargetFramework>
     <CodeGenerationRoslynToolsRootPath>$(OutputPath)..\..\CodeGeneration.Roslyn.Tasks\$(Configuration)\</CodeGenerationRoslynToolsRootPath>
-    <GenerateCodeFromAttributesToolPathOverride>$(OutputPath)..\..\CodeGeneration.Roslyn.Tool\$(Configuration)\netcoreapp1.0\dotnet-codegen.dll</GenerateCodeFromAttributesToolPathOverride>
+    <GenerateCodeFromAttributesToolPathOverride>$(OutputPath)..\..\CodeGeneration.Roslyn.Tool\$(Configuration)\netcoreapp2.0\dotnet-codegen.dll</GenerateCodeFromAttributesToolPathOverride>
   </PropertyGroup>
 
   <ItemGroup>

--- a/src/CodeGeneration.Roslyn.Tool/CodeGeneration.Roslyn.Tool.csproj
+++ b/src/CodeGeneration.Roslyn.Tool/CodeGeneration.Roslyn.Tool.csproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageType>DotnetCliTool</PackageType>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>dotnet-codegen</AssemblyName>
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>

--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -11,7 +11,7 @@
     <RepositoryUrl>https://github.com/aarnott/CodeGeneration.Roslyn</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <IsPackable Condition=" $(MSBuildProjectName.Contains('Test')) ">false</IsPackable>
-    <NerdbankGitVersioningVersion>2.1.23</NerdbankGitVersioningVersion>
+    <NerdbankGitVersioningVersion>2.2.13</NerdbankGitVersioningVersion>
   </PropertyGroup>
 
   <ItemGroup>

